Sources/SubscriberAttributes/SubscriberAttributesManager.swift

```diff
@@ -214,10 +214,8 @@ class SubscriberAttributesManager {
         self.lock.perform {
             var unsyncedAttributes = self.unsyncedAttributesByKey(appUserID: appUserID)
 
-            for (key, attribute) in attributesToSync {
-                if unsyncedAttributes[key]?.value == attribute.value {
-                    unsyncedAttributes[key]?.isSynced = true
-                }
+            for (key, attribute) in attributesToSync where unsyncedAttributes[key]?.value == attribute.value {
```

Comment:
this is pretty cool, I didn't know you could do it

Reply:
I think it's kinda silly, way to make the language more complex for no reason 😅